README.md

```diff
@@ -72,3 +102,21 @@ ubuntu@build-snapcraft-io:~/work$ npm test
 
 Emacs is confused if 'build.snapcraft.io' is used in the project directory
 name, using 'build-snapcraft-io' restores sanity.
+
+## Updating dependencies
+
+Managing dependencies with npm can be a little .... interesting, especially if
```

Comment:
As much as npm can be a pain sometimes we probably don't need opinions in README.